### PR TITLE
Ft/dashboard admin app

### PR DESCRIPTION
**Pull request by Shegsdev**
- Design admin dashboard
- Add scss

https://user-images.githubusercontent.com/35809537/233188606-6df00bf0-36e7-4047-b06a-4eab43113ed2.mp4

### CODE REVIEW

**Review by dejetem — APPROVED** [review]
Good

File: frontend/mms-Admin/components/Layout/SideBar/SideBar.jsx [hunk 1]
```diff
@@ -21,9 +21,9 @@ const SideBar = ({ user }) => {
       onBreakpoint={() => {}}
       onCollapse={(collapsed, type) => {}}>
       <Typography style={{ padding: "32px 55px" }}>
```
Comment (emmsdan) [inline frontend/mms-Admin/components/Layout/SideBar/SideBar.jsx:23]: c

File: frontend/mms-Admin/components/admin/dashboard/Reports.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { Icon } from "components/Icon/Icon";
+import styles from "styles/admin/dashboard.module.css";
+
+const Reports = ({ reports }) => {
+  const { Paragraph } = Typography;
+  return (
+    <div className={styles.card_container}>
+      <Row justify={"space-between"}>
+          <Paragraph>
+            <p style={{ margin: 0 }} className={styles.card_header_text}>Reports overview</p>
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Reports.js:12]: no inline style

File: frontend/mms-Admin/components/admin/dashboard/Reports.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          </Paragraph>
+      </Row>
+      <Row justify={"space-between"} gutter={[16, 16]}>
+        {reports?.map((report) => {
+          return (
+            <Col md={8} sm={12} xs={24}>
+              <div className={styles.card}>
+                <Icon name="Report" width={25} height={33} color="#058B94" />
+                <Paragraph style={{ marginLeft: "1em", display: "inline-block" }}>
+                  <p style={{ margin: 0, fontSize: "20px" }}>{report.title}</p>
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Reports.js:25]: no inline style

File: frontend/mms-Admin/components/admin/dashboard/Statistics.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+import styles from "styles/admin/dashboard.module.css";
+
+const Statistics = ({ stats }) => {
+  const { Title, Paragraph } = Typography;
+  return (
+    <div className={styles.stats}>
+      <div
+        style={{
+          background: "#058b94",
+          borderRadius: "7px",
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Statistics.js:13]: no inline style

File: frontend/mms-Admin/components/admin/dashboard/Statistics.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+          </div>
+          <Typography
+            style={{
+              display: "flex",
+              justifyContent: "start",
+              alignItems: "end",
+            }}>
+            <Title
+              level={6}
+              style={{ color: "#fff", marginRight: "12px", marginBottom: 0 }}>
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Statistics.js:29]: no inline style

File: frontend/mms-Admin/components/admin/dashboard/Statistics.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+          width: "250px",
+        }}>
+        <div>
+          <div style={{ display: "flex", justifyContent: "end" }}>
+            <Button>View</Button>
+          </div>
+          <Typography
+            style={{
+              display: "flex",
+              justifyContent: "start",
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Statistics.js:24]: no inline style

File: frontend/mms-Admin/components/admin/dashboard/Statistics.js [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+              level={6}
+              style={{ color: "#fff", marginRight: "12px", marginBottom: 0 }}>
+              12
+            </Title>
+            <Paragraph style={{ margin: 0 }}>
+              <p
+                style={{
+                  color: "#f7feff",
+                  margin: 0,
+                  width: "80%",
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Statistics.js:37]: no inline style

File: frontend/mms-Admin/components/admin/dashboard/Statistics.js [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+            </Paragraph>
+          </Typography>
+        </div>
+      </div>
+      <div className={styles.card_container}>
+        <div className={styles.top_cards}>
+          {stats?.map((stat) => (
+            <div className={styles.stats_card} key={stat.title}>
+              <Paragraph style={{ margin: 0 }}>
+                <p style={{ margin: 0, fontSize: "20px" }}>{stat.title}</p>
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Statistics.js:52]: no inline style

File: frontend/mms-Admin/components/admin/dashboard/Tasks.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+          style={{
+            background: "#058b94",
+            borderRadius: "7px",
+            padding: "12px",
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+          }}
+        >
+          <p style={{ fontSize: "20px", color: "#f7feff", margin: 0 }}>In Progress</p>
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Tasks.js:28]: no inline style

File: frontend/mms-Admin/components/admin/dashboard/Tasks.js [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+        </Col>
+        <Col md={20} sm={24}>
+          <Row gutter={[16, 16]}>
+          {tasks?.map((task) => {
+            return (
+              <Col md={8} sm={12} xs={24}>
+                <div className={styles.card}>
+                  <Icon name="Report" width={25} height={33} color="#058B94" />
+                  <Paragraph style={{ marginLeft: "1em", display: "inline-block" }}>
+                    <p style={{ margin: 0, fontSize: "20px" }}>{task.title}</p>
```
Comment (emmsdan) [inline frontend/mms-Admin/components/admin/dashboard/Tasks.js:38]: no inline style

**Review by emmsdan — CHANGES_REQUESTED** [review]
please, remove the inline style and package-lock.json

**Review by feisal-Codes — APPROVED** [review]
Good work

**Review by dejetem — APPROVED** [review]

**Review by emmsdan — APPROVED** [review]

**Pull request merged**